astropy/io/votable/src/tablewriter.c

```diff
@@ -33,17 +33,6 @@ next_power_of_2(Py_ssize_t n)
 }
 
 /******************************************************************************
- * Python version compatibility macros
- ******************************************************************************/
-#if PY_MAJOR_VERSION >= 3
-#  define IS_PY3K
-#endif
-
-#ifndef Py_TYPE
-#  define Py_TYPE(o) ((o)->ob_type)
-#endif
```

Comment:
This macro is always defined for PY3 ?

Reply:
Yes, it's defined in Python2 >= 2.7 and Python3 >= 3.1.